initfiles/sbin/complete-uninstall.sh.in

```diff
@@ -88,24 +135,30 @@ elif [ -e /etc/redhat-release -o -e /etc/SuSE-release ]; then
 fi
 
 echo "Removing Directory - ${path}"
-rm -rf ${path}
+removedir ${path}
+
+if [ -n "${mklink}" -a -n "${lpath}" ] ; then
```

Comment:
The issue is that the rpm -e <packagename>  will go ahead and remove /opt/HPCCSystems but since it doesn't lay down /var/log/HPCCSystems (etc.) it doesn't know about them, and therefore doesn't remove them.  That's why complete-uninstall goes through and specifically has to remove those directories (unless we find out they're symlinks, in which case we just remove the contents of the symlink directory.)